build/CopyFilesToStagingDir.ps1

```diff
@@ -122,6 +122,17 @@ PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.Windows.AppLifecycle.w
 PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.Windows.ApplicationModel.DynamicDependency.winmd $NugetDir\lib\native
 #
 # C#/WinRT Projections
+PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.dll $NugetDir\lib\anycpu\net5.0
+PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.pdb $NugetDir\lib\anycpu\net5.0
```

Comment:
>pdb [](http://example.com/codeflow?start=157&length=3)

we should stop copying pdb

Reply:
Are they needed in the Transport Package for use by integration tests?

References to .pdb

* build\build-mrt.yml
* build\CopyFilesToStagingDir.ps1
* build\ProjectReunion-BuildFoundation.yml
* build\publish-mrt.yml
* build\StripSymbols.ps1
* build\AzurePipelinesTemplates\ProjectReunion-RunHelixTests-Job.yml

If I'm reading this right there what we do with PDBs:
1. Copy from build outdir to Helix
2. Copy to Transport Package
3. Strip symbols
4. Publish symbols to internal+public servers
5. [Future] Integration tests use symbols

#1 is righteous for tests but don't involve the Transport Package
If #5 can work off #4 (without lost of fidelity due to #3) then yes, we can stop adding PDBs to the Transport Package

@BenJKuhn @Scottj1s @kythant is there known work on the radar to change PDB handling? Is PDB handling complete and these references (all, not just the ones in this PR) are obsolete and should be removed?

I'm inclined to leave this as-is for this PR, and defer any PDB changes or cleanup to future PR(s). These 2 new .PDBs are small so no real harm; better handled as an all-up thing independent of Projection support.

Reply:
Hui is right, we shouldn't be including pdbs in transport packages - just baggage. Neither WinUI nor IXP transports include them.  Reunion just drops them on the floor in creating appx and thin nupkgs. ProjectReunion-CheckAPICompliance.yml looks like a work in progress, and might need them at some point.  But it should use symchk to pull them down and both 1) verify public indexing of the symbols; and 2) feed them to APIScan (if necessary).  Ryan Shepherd may be driving auditing of feeder symbol indexing, signing, etc.

Reply:
I'll do a full pass dropping PDBs in a follow up PR